spec/controllers/organisations_controller_spec.rb

```diff
@@ -12,8 +12,5 @@
       get :index
       expect(response.content_type).to eq('application/json')
     end
-
-
   end
-
 end
```

Comment:
Yes, during the live-coding-sessions we didn't put that much attention to spaces and blank lines. This is why I have [fixed all lint errors in the latest commit](https://github.com/alphagov/com-prototype/pull/6/commits/db720509f1e2011fdb17b434536cacffa19d73ce), and I have [create a Trello card](https://trello.com/c/O6wNOS9N/79-run-lint-validator-when-running-rspec-tests) to prevent this situation from happening again.